plugin-enclave-gradle/src/main/kotlin/com/r3/conclave/plugin/enclave/gradle/gramine/GenerateGramineManifest.kt

```diff
@@ -37,18 +56,67 @@ open class GenerateGramineManifest @Inject constructor(objects: ObjectFactory) :
             .single { it.startsWith("Location: ") }
             .substringAfter("Location: ")
 
+        /**
+         * It's possible for a Gramine enclave to launch threads internally that Conclave won't know about!
+         * Because of this, we need to add some safety margin.
+         */
+        val conclaveMaxThreads = maxThreads.get()
+        val gramineMaxThreads = conclaveMaxThreads + 8
+
         commandLine(
             listOf(
                 "gramine-manifest",
                 "-Djava_home=${System.getProperty("java.home")}",
                 "-Darch_libdir=/lib/$architecture",
                 "-Dld_preload=$ldPreload",
                 "-Dpython_packages_path=$pythonPackagesPath",
+                "-Dis_python_enclave=${pythonEnclave.get()}",
+                "-Dis_simulation_enclave=${buildType.get() == BuildType.Simulation}",
+                "-Dsimulation_mrsigner=${String(Base64.getEncoder().encode(computeSigningKeyMeasurement()))}",
```

Comment:
Use hex. It's how it's represented in the rest of the code base and thus make things easier to debug.